src/app/components/Button.tsx

```diff
@@ -0,0 +1,27 @@
+import Link from 'next/link';
+import React from 'react';
+import clsx from 'clsx';
+import styles from './Button.module.css';
+
+interface ButtonProps {
+  href: string;
+  label: string;
+  variant: 'green' | 'gray';
+}
+
+export const Button: React.FC<ButtonProps> = ({ href, label, variant }) => {
+  const className = clsx(styles.button, {
+    [styles.greenButton]: variant === 'green',
+    [styles.grayButton]: variant === 'gray',
+  });
+
+  return (
+    <Link href={href}>
+      <span className={className} role="button" tabIndex={0}>
```

Comment:
span を消してもよさそう？

```suggestion
    <Link href={href} className={className}>
```

- このボタンはページ移動することを前提にしてる (=ボタンじゃなくてリンクとして働く) ので `role="button"` は正しくなさそう
  - やるなら `role="link"` だけど a (= Link) タグにしてるから不要
- tabIndex は a タグにはデフォルトになってるっぽいので不要

Reply:
![スクリーンショット 2024-03-15 132914](https://github.com/saitamau-maximum/public-website/assets/132898053/ab0652ab-a001-4a29-b857-c21a7fe4da14)

文字に下線が出るようになったのですが，大丈夫ですか

Reply:
`text-decolation: none`をつけて